config/routes.rb

```diff
@@ -39,6 +39,7 @@
 
     namespace :v2 do
       resources :users
+      resources :providers
       scope :providers do
```

Comment:
Can this be nested inside of the `resource` one?

Reply:
I think @defong is right ... even though I think we should have `/providers`, I think that for the sake of the API we are actually requesting the user's list of providers, which means using `/users/:user_id/providers` to make that explicit.